doc/source/visualizing/plots.rst

```diff
@@ -1054,6 +1055,68 @@ change the property of a single line, give also the index of the profile.
     # change only the first line
     plot.set_line_property("linestyle", "--", 0)
 
+.. _how-to-1d-unstructured-mesh:
+
+1D Line Sampling
+----------------
+
+YT has the ability to sample datasets along arbitrary lines
+and plot the result. You must supply five arguments to the ``LinePlot``
+class. They are enumerated below:
```

Comment:
Is the `ray.profile` method then the one used for plot generation? Probably because I've never used them, I don't see the similarity between rays and `LinePlot`. Rays seem to me like geometric or data objects while `LinePlot` is a plotting class which may obtain its plotting fields from data objects like rays (which is what Nathan's `_sample_ray` method does). I see a closer similarity between `ProfilePlot` and `LinePlot` which I could write to.